deepface/detectors/DetectorWrapper.py

```diff
@@ -149,14 +166,22 @@ def rotate_facial_area(
     Args:
         facial_area (tuple of int): Representing the (x1, y1, x2, y2) of the facial area.
             x2 is equal to x1 + w1, and y2 is equal to y1 + h1
-        angle (float): Angle of rotation in degrees.
-        direction (int): Direction of rotation (-1 for clockwise, 1 for counterclockwise).
+        angle (float): Angle of rotation in degrees. Its sign determines the direction of rotation.
+                       Note that angles > 360 degrees are normalized to the range [0, 360).
         size (tuple of int): Tuple representing the size of the image (width, height).
 
     Returns:
         rotated_coordinates (tuple of int): Representing the new coordinates
             (x1, y1, x2, y2) or (x1, y1, x1+w1, y1+h1) of the rotated facial area.
     """
+
+    # Normalize the witdh of the angle so we don't have to
+    # worry about rotations greater than 360 degrees
+    angle = angle % 360
```

Comment:
angle will be positive always because of the module, so you can get rid of the direction because it will be 1 always

Reply:
That's not right.
75 % 360 == 75
-75 % 360 == -75

Reply:
This is my experiment.

```shell
Python 3.9.16 (main, Jan 11 2023, 16:05:54)
[GCC 11.2.0] :: Anaconda, Inc. on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> a = -75
>>> a % 360
285
>>>
>>>
>>> b = 75
>>> b % 360
```

Reply:
still, both outputs will work as is

never mind this

Reply:
Ouch ... python mod operator is pretty fucked up. Every other language and calculators keeps the sign of dividend not the divisor.

Reply:
![image](https://github.com/serengil/deepface/assets/15280851/eafced3b-2af3-4eb4-9163-f55bf3615f48)